packages/lib/server-only/user/create-user.ts

```diff
@@ -32,6 +39,7 @@ export const createUser = async ({ name, email, password, signature }: CreateUse
       password: hashedPassword,
       signature,
       identityProvider: IdentityProvider.DOCUMENSO,
+      emailVerified,
```

Comment:
added emailVerified arg because this function can be reuseable in user test fixture to create a verified email user